Ladybird/Qt/TaskManagerWindow.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024, Andrew Kaster <akaster@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TaskManagerWindow.h"
+#include <LibWebView/ProcessManager.h>
+#include <QVBoxLayout>
+
+namespace Ladybird {
+
+void show_task_manager_window()
+{
+    auto& window = TaskManagerWindow::the();
+    window.show();
+    window.activateWindow();
+    window.raise();
+}
+
+void close_task_manager_window()
+{
+    TaskManagerWindow::the().close();
+}
+
+TaskManagerWindow::TaskManagerWindow()
+    : m_web_view(new WebContentView(this, {}, {}))
+{
+    setLayout(new QVBoxLayout);
+    layout()->addWidget(m_web_view);
+
+    setWindowTitle("Task Manager");
+    resize(400, 300);
+
+    m_update_timer.setInterval(1000);
+
+    QObject::connect(&m_update_timer, &QTimer::timeout, [this] {
+        this->update_statistics();
+    });
+}
+
+TaskManagerWindow& TaskManagerWindow::the()
+{
+    static TaskManagerWindow instance;
```

Comment:
Can we have someone actually own the TM widgets (here and the next commit)? making everything a singleton feels not great

Reply:
I'm not sure *who* would own it, though. I suppose optimizing for the multi-window world is premature. I can stuff it on BrowserWindow. or perhaps into main? on the Application object? I really don't want to create the object if no one needs it though